GameServer/Logic/GameObjects/CollisionHandler.cs

```diff
@@ -62,14 +64,14 @@ public void addObject(GameObject obj)
         {
             if (divisionCount == -1) // If we have not initialised..
             {
-                Logger.LogCoreError("Tried to add an object before we initialised the CollisionHandler!");
+                _logger.LogCoreError("Tried to add an object before we initialised the CollisionHandler!");
                 return;
             }
 
             var map = obj.GetGame().GetMap();
             if (map != null && map != chart)
             {
-                Logger.LogCoreInfo("Map is adding an object that is not healthy. His map pointer is " + obj.GetGame().GetMap() + " (not " + chart + "). Not adding it.");
+                _logger.LogCoreInfo("Map is adding an object that is not healthy. His map pointer is " + obj.GetGame().GetMap() + " (not " + chart + "). Not adding it.");
```

Comment:
120 char limit exceeds here as well, not your fault I know but please refactor :)